source/tmpl/settings.gohtml

```diff
@@ -45,6 +45,7 @@
                                     <span class="switch">
                                         <input type="checkbox" name="update_notify-option" class="switch" id="switch-update_notify"{{if UPDATENOTIFY}} checked{{end}}>
                                         <label for="switch-update_notify" class="mt-2 mt-sm-0"></label>
+                                        <small class="form-text text-muted">Enabling this will send only Messages when the status of a services changes.</small>
```

Comment:
Maybe replace `messages`->`notifications`?

Reply:
Done.

@okainvo sorry for the delayed message. I'm very sorry about that.